src/base/utils/gzip.cpp

```diff
@@ -29,6 +29,8 @@
 
 #include "gzip.h"
 
+#include <QtGlobal>
```

Comment:
Why it is not in Qt section? There is no non-Qt includes that depend on it so it is under "regular" rules (should be placed in first subsection of Qt includes).